pkg/kubectl/resource_printer.go

```diff
@@ -1629,7 +1630,7 @@ func printEvent(event *api.Event, w io.Writer, options PrintOptions) error {
 
 // Sorts and prints the EventList in a human-friendly format.
 func printEventList(list *api.EventList, w io.Writer, options PrintOptions) error {
-	sort.Sort(SortableEvents(list.Items))
+	sort.Sort(events.SortableEvents(list.Items))
 	for i := range list.Items {
```

Comment:
@kargakis thanks, will do. I have verified that both get events and describe pod, dont call this function printEventList, not sure where else is this called.